tests/Kernel/CorporateBlocks/CorporateBlocksTestBase.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Kernel\CorporateBlocks;
+
+use Drupal\Tests\oe_theme\Kernel\AbstractKernelTestBase;
+
+/**
+ * Test footer block rendering.
+ */
+class CorporateBlocksTestBase extends AbstractKernelTestBase {
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  public static $modules = [
```

Comment:
This needs to be protected. Fix also please in `AbstractKernelTestBase `

Reply:
Fixed